packages/backend/src/modules/dashboard/Board.ts

```diff
@@ -0,0 +1,36 @@
+import mongoose from 'mongoose'
+
+const { ObjectId } = mongoose.Types
+
+const boardSchema = new mongoose.Schema({
+  _id: ObjectId,
+  title: String,
+  description: String,
+  createdAt: Date,
+  status: String,
```

Comment:
create enum for status: PUBLIC, PRIVATE
see
https://mongoosejs.com/docs/validation.html